auction_keeper/main.py

```diff
@@ -244,26 +244,48 @@ def seq_func(check_func: callable):
             if self.arguments.bid_on_auctions:
                 lifecycle.every(self.arguments.bid_check_interval, self.check_for_bids)
 
+    def auction_notice(self) -> str:
+        if self.arguments.type == 'flip':
+            return "--> Check all urns and kick off new auctions if any" + \
+                   " unsafe urns need to be bitten"
+        else:
+            return "--> Check thresholds in Vow Contract and kick off new" + \
+                  f" {self.arguments.type} auctions once reached"
+
     def startup(self):
         self.approve()
         self.rebalance_dai()
         if self.flapper:
             self.logger.info(f"MKR balance is {self.mkr.balance_of(self.our_address)}")
 
+        notice_string = []
         if not self.arguments.create_auctions:
             logging.info("Keeper will not create new auctions")
+        else:
+            notice_string.append(self.auction_notice())
         if not self.arguments.bid_on_auctions:
             logging.info("Keeper will not bid on auctions")
+        else:
+            notice_string.append("--> Check all auctions being monitored and evaluate" + \
+                                f" bidding opportunity every {self.arguments.bid_check_interval} seconds")
 
         if self.deal_all:
-            logging.info("Keeper will deal auctions for any address")
+            notice_string.append("--> Check all auctions and deal for any address")
         elif len(self.deal_for) == 1:
-            logging.info(f"Keeper will deal auctions for {list(self.deal_for)[0].address}")
+            notice_string.append(f"--> Check all auctions and deal for {list(self.deal_for)[0].address}")
         elif len(self.deal_for) > 0:
-            logging.info(f"Keeper will deal auctions for {self.deal_for} addresses")
+            notice_string.append(f"--> Check all auctions and deal for {self.deal_for} addresses")
         else:
             logging.info("Keeper will not deal auctions")
 
+        if notice_string:
+            logging.info("Keeper will perform the following operation(s) in parallel:")
+            [logging.info(line) for line in notice_string]
+
+            logging.info("*** Although it may seem 'stuck', the Keeper is executing the above processeses and will rest in this state while awaiting new auctions ***")
```

Comment:
I'm uncomfortable with this log message.  What if it is stuck?  If users want more feedback to show their keeper is running, I recommend promoting the log messages on lines 355 and 479 from `debug` to `info`.

Reply:
Nice suggestion. Will promote lines 355 and 479. With all do respect, what is your method for determining if a Keeper is stuck?

Reply:
If it's kicking flip auctions, without using Vulcanize, it could take > 20 minutes to check all the vaults.  This could be perceived as being "stuck", and promoting those logs would give the user some insight.

Reply:
I'm considering changing L285 to:
```
logging.info("*** When Keeper is dealing/bidding, the initial evaluation of auctions will likely take > 45 minutes without setting a lower boundary via '--min-auction' ***")
logging.info("*** When Keeper is kicking, the recurring query of Vaults will likely take > 30 minutes each loop without using VulcanizeDB via `--vulcanize-endpoint` ***")
```
Thoughts before I commit ?

Reply:
You arrived at those measurements experimentally I presume?  Seems reasonable.  We need a better mechanism than the `dead_auctions` array.  I'd like to work on that after gas issues are sorted.

Reply:
Yes